js/src/tab.js

```diff
@@ -168,7 +168,12 @@ class Tab extends BaseComponent {
     event.stopPropagation()// stopPropagation/preventDefault both added to support up/down keys without scrolling the page
     event.preventDefault()
     const isNext = [ARROW_RIGHT_KEY, ARROW_DOWN_KEY].includes(event.key)
-    const nextActiveElement = getNextActiveElement(this._getChildren(), event.target, isNext, true)
+    const nextActiveElement = getNextActiveElement(this._getChildren().filter(nextElement => !isDisabled(nextElement)), event.target, isNext, true)
+
+    if (!nextActiveElement) {
```

Comment:
```suggestion
    if (nextActiveElement) {
       Tab.getOrCreateInstance(nextActiveElement).show()
    }
```

to simple check, so we can use it, in direct :D